scripts/foundation-wallet-deployment/config.ts

```diff
@@ -0,0 +1,59 @@
+import { DeployedContracts } from "../../src-ts/types";
+
+import fs from 'fs'
+
+const goerliDeployedContracts = JSON.parse(
+  fs.readFileSync("./files/goerli/deployedContracts.json").toString()
+) as DeployedContracts;
+const mainnetDeployedContracts = JSON.parse(
+  fs.readFileSync("./files/mainnet/deployedContracts.json").toString()
+) as DeployedContracts;
+
+interface FoundationWalletDeploymentConfig {
+  beneficiary: string;
+  startTimestamp: number;
+  vestingPeriodInSeconds: number;
+  l2ArbitrumGovernor: string;
+  l2GovProxyAdmin: string;
+}
+
+interface ChainIDToConfig {
+  [key: number]: FoundationWalletDeploymentConfig;
+}
+
+const secondsInADay = 60 * 60 * 24;
+
+const secondsInFourYears = secondsInADay * (365 * 4 + 1); // Days in 4 years, includes 1 day for leap day
+
+const chainIDToConfig: ChainIDToConfig = {
+  42161: {
+    beneficiary: "", // TODO
```

Comment:
remaining todo

Reply:
yep, deliberate; foundation will provide the address